scissors/scissors_game/scissors.c

```diff
@@ -4,27 +4,35 @@
 #include <time.h>
 #include "scissors.h"
 
+const int game_table[3][3]={
+	eDRAW,   eLOST,  eWINN,
+	eWINN,   eDRAW,  eLOST,
+	eLOST,   eWINN,  eDRAW
+};
```

Comment:
Such a table is hard for verification without additional comments what each row and column corresponds to.